### PR TITLE
Added Community Section

### LINKED ISSUE #142: Community Section
Add a "Join Our Community" section to the landing page with links to our Discord server and Slack.
Links: [Slack](https://join.slack.com/t/kwoc-koss/shared_invite/zt-wlftnk75-VoQHEEB9WpkHfza6~GGpWQ), [Discord](https://discord.gg/efFwh6fnjk).

Please make a PR to the `landing-page` branch.

### PR DESCRIPTION
**Pull request by jamnxdev**
Fixes #142

### CODE REVIEW

File: src/components/JoinOurCommunity.tsx [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+export default function JoinOurCommunity() {
+    return (
+        <>
+            <h1 className="text-zinc-300 text-center mb-4 text-4xl font-extrabold leading-none tracking-tight md:text-5xl mt-36 lg:text-8xl">
+                Join Us
+            </h1>
+            <p className="text-zinc-300 text-center text-lg font-normal lg:text-2xl px-10 sm:px-16 xl:px-96 mt-10">
+                Community Description
+            </p>
```
Comment (rajivharlalka) [inline src/components/JoinOurCommunity.tsx]: Why is this paragraph tag needed?

Comment (jamnxdev) [reply src/components/JoinOurCommunity.tsx]: Description/brief about community can be added if there is any, I will remove it if its not needed.

Comment (rajivharlalka) [reply src/components/JoinOurCommunity.tsx]: @harshkhandeparkar  is this needed?

Comment (harshkhandeparkar) [reply src/components/JoinOurCommunity.tsx]: I don't think it is necessary but we can write a one liner like Join for future updates, queries and to make connections...

File: src/assets/slack_logo.png
Comment (harshkhandeparkar) [inline src/assets/slack_logo.png:1]: This logo looks stretched. Can you keep the original dimensions? Alternatively, you can just keep the logomark and write the words "Slack Workspace" and "Discord Server" in html.

File: src/components/JoinOurCommunity.tsx [hunk 2]
```diff
@@ -0,0 +1,45 @@
+import DiscordLogo from "../assets/discord_logo.png";
+import SlackLogo from "../assets/slack_logo.png";
+
+export default function JoinOurCommunity() {
+    return (
+        <>
+            <h1 className="text-zinc-300 text-center mb-4 text-4xl font-extrabold leading-none tracking-tight md:text-5xl mt-36 lg:text-8xl">
+                Join Us
```
Comment (harshkhandeparkar) [inline src/components/JoinOurCommunity.tsx]:
```suggestion
                Join Our Community
```

**Review by harshkhandeparkar — CHANGES_REQUESTED** [review]

**Review by harshkhandeparkar — COMMENTED** [review]
Can you make the buttons smaller? Or add some text? They look empty. Also don't worry about the conflicts, I will resolve them.

**Comment by jamnxdev** [comment]
> Can you make the buttons smaller? Or add some text? They look empty. Also don't worry about the conflicts, I will resolve them.

Sure, I will first replace the text with something like "click here to join" and if it doesn't fit the UI, I'll simply reduce the button size.

**Comment by harshkhandeparkar** [comment]
> > Can you make the buttons smaller? Or add some text? They look empty. Also don't worry about the conflicts, I will resolve them.
> 
> Sure, I will first replace the text with something like "click here to join" and if it doesn't fit the UI, I'll simply reduce the button size.

No, don't increase the button text, try to add some description. If that is not possible, just reduce the card size. Maybe just reduce it to a card (button) with the logo and the "Discord Server" title. Clicking the entire card leads to the invite link.

**Review by harshkhandeparkar — COMMENTED** [review]
Can you add the hover effect to the cards as the previous button?

**Review by harshkhandeparkar — APPROVED** [review]

File: src/assets/discord_logo.png
Comment (rohan-b-84) [inline src/assets/discord_logo.png:1]: use images of appropriate size

File: src/assets/slack_logo.png
Comment (rohan-b-84) [inline src/assets/slack_logo.png:1]: Here as well

**Review by rohan-b-84 — DISMISSED** [review]

File: src/assets/discord_logo.png
Comment (harshkhandeparkar) [reply src/assets/discord_logo.png:1]: Use svgs if they are available.

**Review by harshkhandeparkar — APPROVED** [review]

**Comment by harshkhandeparkar** [comment]
Thank you @Jaimin25! :rocket:

**Pull request merged**